components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImpl.java

```diff
@@ -2921,5 +2929,52 @@ private ArrayList<String> getLocalClaimURIs(List<LocalClaim> localClaims) {
         }
         return localClaimsArray;
     }
-}
 
+    private void postApplicationUSerAttributeUpdate(ServiceProvider updatedApp, ServiceProvider storedApp,
+                                                    String tenantDomain) {
+
+        if (updatedApp.getClaimConfig().getClaimMappings().length > 0) {
```

Comment:
This can not be null https://github.com/wso2/carbon-identity-framework/blob/df08943028dc40d9b6470fbdc568f2b18d245e35/components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ClaimConfig.java#L60